composer.json

```diff
@@ -1,11 +1,13 @@
 {
   "license": "GPL-3.0+",
   "require": {
+    "ext-intl": "*",
```

Comment:
Yes, it's for NumberFormatter. Composer can't install extensions but it does alert people to the fact that they are required.